starter-code/css/main.css

```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    font-family: 'Montserrat', 'sans-serif';
+}
+
+nav {
+    position: fixed;
+    border-color: white;
+    overflow: visible; 
+}
+
+header .spotify-green-logo {
+    float: left;
+    height: 15%;
+    width: 15%;
+    margin-right: 65em;
+}
```

Comment:
Because the spotify logo is the only `img` in the header, you can target this using `header img` or make `spotify-green-logo` an `id`